src/flowgger/fuzzer.rs

```diff
@@ -0,0 +1,210 @@
+extern crate quickcheck;
+
+use crate::flowgger;
```

Comment:
if this is in source file, and only used for test, make it clear. e.g. like in [test_utils](https://github.com/awslabs/flowgger/blob/master/src/flowgger/utils/test_utils.rs#L1): prefix the file with `test_` to make it clear it is not a source file. and don't make any import outside the test clause to make sure nothing is imported